components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -595,9 +595,7 @@ public DataSection documentInfoHeader(Set<String> enabledFields) throws Exceptio
 
     public DataSection documentInfoBody(Patient patient) throws Exception
     {
-        @SuppressWarnings("deprecation")
-        DocumentAccessBridge dab = Utils.getComponent(DocumentAccessBridge.class);
-        XWikiDocument patientDoc = (XWikiDocument) dab.getDocument(patient.getDocument());
+        XWikiDocument patientDoc = patient.getXDocument();
```

Comment:
Useless import of [DocumentAccessBridge](https://github.com/phenotips/phenotips/blob/5bf1c40014f2e5de26835a4dcd71fc0803b75161/components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java#L30) was not removed.
You can configure your IDE to automatically remove unused imports. This simply saves costly developers time and code-reviewers time.